framework/lib/components/toolbox/toolbox.tsx

```diff
@@ -72,13 +83,47 @@ export const Toolbox = ({
           in={ isOpen }
           style={ {
             height: container.h as string,
-            width: toolboxWidth,
+            width: adjustableWidth,
             zIndex: coreZIndex.overlay,
           } }
         >
-          <Flex sx={ { ...container, w: toolboxWidth } } { ...rest } onKeyDown={ handleKeyDown }>
+          <Flex
+            sx={ { ...container, w: adjustableWidth } }
+            { ...rest }
+            onKeyDown={ handleKeyDown }
+            style={ {
+              position: 'relative',
+              overflow: 'hidden',
+            } }
+          >
+            <div
+              style={ {
+                cursor: 'ew-resize',
+                position: 'absolute',
+                top: 0,
+                left: 0,
+                width: '10px',
+                height: '100%',
+                zIndex: 2,
+                userSelect: 'none',
+              } }
+              onMouseDown={ (e) => {
+                e.preventDefault() // prevents text from being marked on drag
+                const startX = e.clientX
+                const onMouseMove = (event: { clientX: number }) => {
+                  const newWidth = adjustableWidth - (event.clientX - startX)
+                  setAdjustableWidth(Math.max(newWidth, 100)) // set minimum width to 100px
```

Comment:
Put a max width here as well, it should be the width of the size ="lg", the min width should also be the width of size="sm", the comment can be removed

Reply:
@SebCodesTheWeb , I have done this now, but are we sure we want it like that? As you can see, there are really a small difference between sm and lg. I think it makes the resize functionality not very usable when we can resize it in such a small range. Movie below
https://github.com/mediatool/northlight/assets/81630176/409d0d3c-02c6-4895-9e65-3557382e842c